data/battle_anim_815A0D4.s

```diff
@@ -505,6 +505,5 @@ gUnknown_085CE9B0:: @ 85CE9B0
 	.align 2
 gUnknown_085CE9C8:: @ 85CE9C8
 	.2byte 0x0000, 0x0010, 0x0400, 0x0000, 0xfffe, 0x0000, 0x0800, 0x0000, 0x0000, 0x0004, 0x0400, 0x0000, 0x0000, 0x0000, 0x1800, 0x0000
-	.2byte 0x0001, 0xfffb, 0x1000, 0x0000, 0x7fff, 0x0000, 0x0000, 0x0000, 0x03e0, 0x001f, 0x001c, 0x001a, 0x0018, 0x0000, 0x0000, 0x0000
-	.2byte 0x0000, 0x0000, 0x0000, 0x0000, 0x0000, 0x6318, 0x0000, 0x7fff
```

Comment:
what was up with this change getting shorter again?

Reply:
That was a palette that was part of learn_move